_pages/software-and-tools/github.md

```diff
@@ -110,7 +110,11 @@ Contractors or external government collaborators should only be added to teams w
 
 ### Archiving
 
-As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date. Inactive repositories are automatically archived via [ghad](https://github.com/18F/ghad).
+As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date.
+
+Repositories created as part of an engagement with a partner should be transferred to the partner agency, forked back to [one of our organizations](#organizations), and then archived. See the [handoff to partner agencies](#handoff-to-partner-agencies) section for more information.
+
+Inactive repositories are automatically archived by the TTS Tech Portfolio with [ghad](https://github.com/18F/ghad). A repository is considered "inactive" if there haven't been any new commits or comments on issues or pull requests in a while; or if the repository description includes the terms "deprecated," "not supported," or "no longer supported." The [current default is 90 days](https://github.com/18F/ghad/blob/main/cli.js#L37). For repositories that are still maintained and used but have infrequent commits or comments, maintainers can add the `MAINTAINED` topic to the repository to exempt it from automatic archiving.
```

Comment:
> maintainers can add the `MAINTAINED` topic to the repository to exempt it from automatic archiving

I'd consider myself an above-average GH user, but when I read this line I didn't really know what this would look like in practice. Could we link to a resource explaining how to add a topic to a repository?

Reply:
Oh, that is a good thing to clarify! Even as I was writing it, I was like... "Are people going to know what the heck this means?" And now I have an answer.